installer/resources/pacbot_app/files/DB_Policy.sql

```diff
@@ -3528,4 +3528,9 @@ UPDATE cf_PolicyTable SET severity='critical' WHERE policyId='Enable_Secure_Boot
 UPDATE cf_PolicyTable SET severity='high' WHERE policyId='Enable_VPC_Flow_Logs_and_IntraNode_Visibility';
 
 UPDATE cf_PolicyTable SET policyDisplayName = 'Set NSG flow log retention period >= N days' WHERE policyId ='Network_Security_Group_Flow_Log_retention_more_than_90_days';
-UPDATE cf_PolicyTable SET policyDisplayName='Restrict Inbound Traffic to RDP Port 3389' WHERE policyId='NaclPublicAccessPort_version-1_NACLWithPublicAccessForPort3389_networkacl';
\ No newline at end of file
+UPDATE cf_PolicyTable SET policyDisplayName='Restrict Inbound Traffic to RDP Port 3389' WHERE policyId='NaclPublicAccessPort_version-1_NACLWithPublicAccessForPort3389_networkacl';
+
+INSERT IGNORE INTO cf_PolicyParams (`policyID`, `paramKey`, `paramValue`, `defaultVal`, `isEdit`, `isMandatory`, `encrypt`, `displayName`, `description`) VALUES ('IamAccessKeyRotatedInEvery90Days_version-1_IamAccessKeyRotatedInEvery90Days_iamuser','rotationDuration','90','90','true','true','false','Key rotation duration','Specify duration in days for access key rotation');
```

Comment:
@ershad-paladin is it difficult to standardize the ids?

We have names that are synonyms for which we have:

In90days
GreaterThan90
More_than_ninety

If possible please come up with a common notation for the keys.

Reply:
we have 6 policies for N days. For policyId change, I'd suggest to create a separate ticket and address all these policiesIds in that. PFB all the policyIds.

CheckInactiveIamUser_version-1_CheckInactiveIamUser_iamuser
Ec2StoppedInstanceForLong_version-1_Ec2StoppedInstanceForLong_ec2
IamAccessKeyRotatedInEvery90Days_version-1_IamAccessKeyRotatedInEvery90Days_iamuser
Network_Security_Group_Flow_Log_retention_more_than_90_days
Set_Retention_Days_Greater_Than_Ninety
kms_keys_rotation_period_within_90_days

Reply:
If the policy id change is not a breaking change let's do it in this ticket.

Reply:
done